packages/flame/lib/src/components/steerable_component.dart

```diff
@@ -0,0 +1,117 @@
+import 'dart:math' as math;
+
+import 'package:meta/meta.dart';
+
+import '../ai/steer/steerable.dart';
+import '../ai/steer/steering_acceleration.dart';
+import '../ai/steer/steering_behavior.dart';
+import '../anchor.dart';
+import '../extensions/vector2.dart';
+import 'position_component.dart';
+
+class SteerableComponent extends PositionComponent implements Steerable {
+  SteerableComponent({
+    Vector2? velocity,
+    double? angularSpeed,
```

Comment:
`angularVelocity`?

Reply:
_Velocity_ is a vector quantity, whereas _speed_ is scalar. Whereas in 3d space rotation would be represented as a vector, in 2d rotation is a simple scalar, and hence its rate of change should be called speed.

Reply:
Speed doesn't have a direction though, and I guess this can be negative?

Reply:
hmm, fair point. I guess i can rename it into velocity then